packages/dev/docs/pages/error.mdx

```diff
@@ -0,0 +1,29 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import {Content} from '@react-spectrum/view';
+ import {Flex} from '@react-spectrum/layout';
+import {Heading} from '@react-spectrum/text';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import NotFound from '@spectrum-icons/illustrations/NotFound';
+
+---
+title: Page not found
+---
+
+<Flex alignItems="center" direction="row" height="50vh" justifyContent="center">
+  <IllustratedMessage>
+    <NotFound />
+    <Heading>Error 404: page not found</Heading>
+    <Content>Try reloading this page. If the error persists, contact the React Spectrum team to report it.</Content>
```

Comment:
I'm not a huge fan of this text; if someone has suggestions im all ears

Reply:
is it for errors? or only 404?
We couldn't find your page? Something went wrong?
Do we even bother saying reload anymore?

Reply:
Can we put a link for where to contact us?

Reply:
good point -- reloads wont matter. its just a 404.

Reply:
Maybe we say something about choosing a link in the left nav?

Reply:
You could use the text Spectrum uses in the contribution? https://spectrum-contributions.corp.adobe.com/page/illustrated-message-beta/#Error:-404

```suggestion
    <Content>This page isn't available. Try checking the URL or visit a different page.</Content>
```